node/node.go

```diff
@@ -753,8 +753,8 @@ func makeNodeInfo(
 	if _, ok := txIndexer.(*null.TxIndex); ok {
 		txIndexerStatus = "off"
 	}
-	nodeInfo := p2p.NodeInfo{
-		ID:      nodeID,
+	nodeInfo := p2p.DefaultNodeInfo{
+		ID_:     nodeID,
```

Comment:
Is there a better alternative? I really don't like `GetID()` for this.

Reply:
I saw `Num` suffix somewhere in our codebase